src/tests/specs/quickPreview.spec.ts

```diff
@@ -37,6 +37,16 @@ describe('Quick preview tests: ', () => {
     expect(await planner.quickPreview.getLabels()).toContain(c.newLabel);
   });
 
+  it('should open quickpreview and create new label', async () => {
```

Comment:
@sahil143 please change it statement, there's already one tests with the same name